projects/demo/src/modules/app/demo-routes.ts

```diff
@@ -59,7 +59,7 @@ export const DemoRoute = {
     Expand: '/components/expand',
     ElasticContainer: '/components/elastic-container',
     FieldError: '/pipes/field-error',
-    FormLayout: '/components/form',
+    FormLayout: '/layout/form',
```

Comment:
Let's remove old Form page completely and rename this to `Form`. Apparently old form page is visited very often and we do not want people to copy paste deprecated example.

Reply:
Only layouts form page should exist in demo right?